scripts/Muon/GUI/ElementalAnalysis/PeriodicTable/periodic_table_view.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import
+
+from PyQt4 import QtCore, QtGui
+
+from silx.gui.widgets import PeriodicTable
+
+
+class PeriodicTableView(QtGui.QWidget):
+    sig_table_changed = QtCore.pyqtSignal(list)
```

Comment:
I think we just put `object` instead of `list` etc.

Reply:
Agree with this - just tested and works fine